lisa/nic.py

```diff
@@ -333,9 +333,9 @@ def _get_node_nic_info(self) -> None:
                     upper_nic_mac = ip.get_mac(nic_name)
                     if upper_nic_mac == lower_nic_mac:
                         upper_nic = nic_name
+                        nic_info = NicInfo(upper_nic, lower_nic, pci_slot)
+                        self.append(nic_info)
                         break
```

Comment:
Does it need to return exception, if it's not found?

Reply:
No. It essentially means that SRIOV is not present.